docs/asyncapi.yaml

```diff
@@ -0,0 +1,92 @@
+asyncapi: 3.0.0
+info:
+  title: FCP-FD-COMMS
```

Comment:
Good question! I do agree that it would be better to be descriptive. I would go for something like : "API contract book for Single Front Door customer contacts"